package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "oauth-jwt-node",
+  "type": "module",
+  "dependencies": {
+    "dotenv": "^16.4.5",
+    "express": "^4.18.2",
+    "jsonwebtoken": "^9.0.2"
+  },
+  "devDependencies": {
+    "eslint": "^8.56.0",
+    "prettier": "^3.2.5",
+    "vitest": "^1.3.1"
+  },
+  "scripts": {
+    "start:auth-server": "node ./auth-server/authServer.js &",
+    "start:service": "node ./service/service.js &",
+    "start:auth-server:watch": "node --watch ./auth-server/authServer.js",
+    "start:service:watch": "node --watch ./service/service.js",
+    "client": "node ./client/client.js",
+    "test": "npm run start:service && npm run start:auth-server && vitest run && npm run kill:service && npm run kill:auth-server",
+    "kill:service": "pkill -f ./service/service.js",
+    "kill:auth-server": "pkill -f ./auth-server/authServer.js'"
```

Comment:
OK. But two things:

1. Notice that if you use pkill your are killing the process by name. If you have two sessions running the same tests or the servers up for developmet you'll have undesired results
2.   It's better if the test suite that spins up the services and stops them as needed. The test suite can store the pid of the forked server and kill it by pid.

Reply:
I didn't realize that could happen. Next time this could be a good practice to do, thanks.